requirements/base.txt

```diff
@@ -26,3 +26,7 @@ easy-thumbnails[svg]==2.8.5
 jsonfield==3.1.0
 python-dateutil==2.8.2
 rules==3.3
+
+# celery
+celery==5.3.1
+redis==5.0.0
```

Comment:
should be moved to a4 since a4 has also background task
see: https://github.com/liqd/adhocracy-plus/blob/main/requirements/base.txt#L16-L20

Reply:
done, see https://github.com/liqd/adhocracy4/pull/1488